inference-engine/src/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -114,7 +114,6 @@ bool ngraph::pass::CommonOptimizations::run_on_function(std::shared_ptr<ngraph::
     common_fusions->add_matcher<ngraph::pass::ShuffleChannelsFusion>(false);
     common_fusions->add_matcher<ngraph::pass::SpaceToBatchFusion>();
     common_fusions->add_matcher<ngraph::pass::BatchToSpaceFusion>();
-    common_fusions->add_matcher<ngraph::pass::TransposeToReshape>();
```

Comment:
Regression for old IRs

Reply:
No, as we still call this transformation inside MOC pipeline which is included into CommonOptimization pipeline, so nothing is going to change for old IRs. But for new IRs we will perform this transformation in offline so in runtime this transformation will do nothing.